src/main/webapp/WEB-INF/tags/helpPage.tag

```diff
@@ -31,7 +30,9 @@
     <div id="footerComponent" class="container-fluid">
         <div class="container">
             <div class="row">
-                <t:navBarBrand/>
```

Comment:
Aren't the two of them different components? Line ~34 is for the footer, while Line 23 is for the header? O_o